src/Components/SelectedMovie/SelectedMovie.js

```diff
@@ -1,14 +1,20 @@
 import React from 'react'
 import './SelectedMovie.css'
 
+function SelectedMovie({selectedMovie, setMovieView, setSelectedMovie}){
 
-function SelectedMovie({selectedMovie}){
   return (
-    <div className='selectedMovie'>
-      <img className='selectedimg' src={selectedMovie.backdrop_path}></img>
-      <h3 className='selectedTitle'>{selectedMovie.title}</h3>
-      <p className='selectedReleaseDate'>Release date: {selectedMovie.release_date}</p>
-    </div>
+  <div 
+  className='selectedMovie'
+  style={{ backgroundImage: `url(${selectedMovie.backdrop_path})` }}
+  >
+    <h3 className='selectedTitle'>{selectedMovie.title}</h3>
+    <p className='selectedReleaseDate'>Release date: {selectedMovie.release_date}</p>
```

Comment:
Note for future selves: if we have time we can do some data manipulating to the release date to make it more humanly readable i.e. "March 5, 2023"

Reply:
We can do more rendering and styling with additional data given, overview, tagline, trailer, etc

Reply:
My suggestion is to try to avoid using inline styling as much as possible, as it can make our code appear messy and difficult to read.